requirements.txt

```diff
@@ -0,0 +1,18 @@
+beautifulsoup4==4.4.1
+chardet==2.3.0
+click==6.6
+dj-database-url==0.4.1
+Django==1.10.2
+Flask==0.11.1
+gunicorn==19.6.0
+html5lib==0.999
+itsdangerous==0.24
+Jinja2==2.8
+logilab-common==1.1.0
+lxml==3.5.0
+MarkupSafe==0.23
+psycopg2~=2.6.2
+requests==2.11.1
+six==1.10.0
+Werkzeug==0.11.11
+whitenoise==3.2.2
```

Comment:
https://docs.travis-ci.com/user/environment-variables/#Defining-public-variables-in-.travis.yml may help.